tests/test_project.cairo

```diff
@@ -119,6 +120,27 @@ fn default_setup() -> (ContractAddress, EventSpy) {
     (project_address, spy)
 }
 
+/// Mint shares without the minter contract. Testing purposes only.
```

Comment:
might want to add a test utils.cairo to avoid code repetition

Reply:
Yes indeed, will be done in next test PR